src/actions/keys.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const alt = require('../alt');
+
+class KeyActions {
+  findNext(evt) {
+    this.dispatch(evt);
+  }
+  findPrevious(evt) {
+    this.dispatch(evt);
+  }
+  moveByScrollUpLine(evt) {
+    this.dispatch(evt);
+  }
+  moveByScrollDownLine(evt) {
+    this.dispatch(evt);
+  }
+  tab(evt) {
```

Comment:
name this indent (?) and move to own action class (does it make sense to group with any other actions?)